django-stubs/views/decorators/http.pyi

```diff
@@ -11,6 +11,6 @@ require_GET: Callable[[_F], _F]
 require_POST: Callable[[_F], _F]
 require_safe: Callable[[_F], _F]
 
-def condition(etag_func: Callable | None = ..., last_modified_func: Callable | None = ...) -> Callable: ...
+def condition(etag_func: Callable | None = ..., last_modified_func: Callable | None = ...) -> Callable[[_F], _F]: ...
```

Comment:
Can you please also type `etag_func` and `last_modified_func` while we are at it? https://docs.djangoproject.com/en/4.2/topics/conditional-view-processing/#the-condition-decorator

Reply:
Yes, good idea. I've updated both according to the docs.
Also, in the same commit, changed sensitive variables and post parameters from `Any` to `str` because that's what Django expects.